sc/shared/src/main/scala/sigmastate/lang/SigmaTyper.scala

```diff
@@ -19,6 +18,7 @@ import scala.collection.mutable.ArrayBuffer
   */
 class SigmaTyper(val builder: SigmaBuilder,
```

Comment:
ScalaDoc (class args description) is missed

Reply:
done